Src/MoneyFox.Windows.LiveandLockScreenTiles/Class1.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.AppService;
+using Windows.ApplicationModel.Background;
+using Windows.Foundation.Collections;
+
+namespace MoneyFox.Windows.LiveandLockScreenTiles
+{
+    public sealed class UpdateliveandLockscreenTiles : IBackgroundTask
```

Comment:
I think the file name was not adjusted here. That one seems still to be on "Class1.cs"

Reply:
On a further look, is it possible that this is an obsolet project? I saw that you added a "UpdateliveandLockscreenTiles " to the Tasks project.

Reply:
i thought when i deleted that from the project after moving to task that i would not display, it is obsolet project that was deleted  so not sure why it still is displaying

Reply:
I assume you clicked remove on the project in Visual Studio? In that case it only removes the project from the solution but the folder and all the files are still there. You can just go to the file location of the repo on your computer and delete that folder. :)